src/AWS.Deploy.Common/DeploymentManifest/DeploymentManifestModel.cs

```diff
@@ -11,11 +11,14 @@ namespace AWS.Deploy.Common.DeploymentManifest
     /// </summary>
     public class DeploymentManifestModel
     {
+        public List<LastDeployedStack> LastDeployedStacks { get; set; }
```

Comment:
JSON property names are inconsistent. This one is `LastDeployedStacks` and deployment projects is `deployment-projects`. When should use either pascal casing or hyphen across the file.